python/cudf/cudf/_lib/strings/CMakeLists.txt

```diff
@@ -12,27 +12,7 @@
 # the License.
 # =============================================================================
 
-set(cython_sources
-    attributes.pyx
-    capitalize.pyx
-    case.pyx
-    char_types.pyx
-    combine.pyx
-    contains.pyx
-    extract.pyx
-    find.pyx
-    find_multiple.pyx
-    findall.pyx
-    json.pyx
-    padding.pyx
-    repeat.pyx
-    replace.pyx
-    replace_re.pyx
-    strip.pyx
-    substring.pyx
-    translate.pyx
-    wrap.pyx
-)
+set(cython_sources)
```

Comment:
I think if there is no more Cython source, we can delete lines 15-22.

Reply:
Ah great. Yes can remove this here